examples/spark-app-demo/emr/configs/emr_cloudformation_template.yml

```diff
@@ -0,0 +1,55 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Missing license here. Make sure to run tests locally.

Reply:
ignore it